x/staking/client/cli/tx.go

```diff
@@ -122,7 +123,7 @@ func NewEditValidatorCmd() *cobra.Command {
 			if minSelfDelegationString != "" {
 				msb, ok := sdk.NewIntFromString(minSelfDelegationString)
 				if !ok {
-					return types.ErrMinSelfDelegationInvalid
+					return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, "minimum self delegation must be a positive integer")
```

Comment:
you can use the wrap method of the error, it makes the code a bit shorter

```
sdkerrors.ErrInvalidRequest.Wrap("minimum self delegation must be a positive integer"
```

Same in other places.

Reply:
Thanks for bringing this to my attention, I'll take a look at the `Wrap()` implementation. I'll refactor these accordingly.

Reply:
I think this is more of a nit. We use the design currently coded throughout the repo and the one robert mentioned in a few places. 

@robert-zaremba want to approve this?

Reply:
I'm not blocking it  - I only made a suggestion. The PR already got enough approvals.

Reply:
BTW - we implemented the `Wrap` method for the error types 2-3 months ago, that why it's not use that much.